src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +74,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      DataAccess.instance.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Maybe the following would be a bit less noisy as clients could just do `DataAccess().updateExecutionBackendInfo(...)`?

```scala
object DataAccess {
  def apply(): DataAccess = instance

  lazy val instance = new slick.SlickDataAccess()
}
```